src/main/java/com/danliuk/validation/TicketValidator.java

```diff
@@ -0,0 +1,42 @@
+package com.danliuk.validation;
+
+import com.danliuk.model.BusTicket;
+import lombok.Getter;
+
+import java.time.LocalDate;
+
+public class TicketValidator {
+    @Getter
+    private static int invalidDate = 0;
+    @Getter
+    private static int invalidType = 0;
+    @Getter
+    private static int invalidPrice = 0;
+    @Getter
+    private static int totalInvalid = 0;
+
+    public static void validateTicket(BusTicket busTicket){
+        int a = 0;
```

Comment:
easier to make it with boolean flag
cause you can just set false every time you detect incorrect field value without any additional checks (like if (a == 0))